cdtb/tftdata.py

```diff
@@ -291,6 +292,9 @@ def parse_champs(self, map22, traits):
                 rarity = champ.getv("mRarity", 0) + 1
                 cost = rarity + int(rarity / 6)
 
+            role_key = record.getv("CharacterRole", None)
+            role = role_entries[role_key].getv("name", None) if role_key is not None else None
```

Comment:
```suggestion
            role_key = record.getv("CharacterRole")
            role = role_entries[role_key].getv("name") if role_key is not None else None
```
Returning `None` is the default behavior and not explicited in other parts of the code.